cla-backend/cla/models/github_models.py

```diff
@@ -962,16 +959,19 @@ def get_pull_request_commit_authors(pull_request) -> List[UserCommitSummary]:
                 # committter different from the author
                 if commit.committer:
                     if commit.committer.id != commit.author.id:
-                        commit_author_summary = UserCommitSummary(
-                            commit.sha,
-                            commit.committer.id,
-                            commit.committer.login,
-                            commit.committer.name,
-                            commit.committer.email,
-                            False, False  # default not authorized - will be evaluated and updated later
-                        )
-                        cla.log.debug(f'{fn} - Committer PR: {pull_request.number}, {commit_author_summary}')
-                        commit_authors.append(commit_author_summary)
+                        if 'web-flow' not in commit.committer.login:
+                            commit_author_summary = UserCommitSummary(
+                                commit.sha,
+                                commit.committer.id,
+                                commit.committer.login,
+                                commit.committer.name,
+                                commit.committer.email,
+                                False, False  # default not authorized - will be evaluated and updated later
+                            )
+                            cla.log.debug(f'{fn} - Committer PR: {pull_request.number}, {commit_author_summary}')
+                            commit_authors.append(commit_author_summary)
+                        else:
+                            cla.log.debug(f'{fn} - Skipping web-flow user: {commit.committer.login}')
```

Comment:
@wanyaland can you please add the comment to the code explaining what is `web-flow` user and why we are skipping it. Thank you!